src/core_test.tsx

```diff
@@ -95,12 +95,13 @@ describe('app', () => {
 
   describe('emit()', () => {
     const update = [Msg, always({})];
+    const state = { emit: identity, relay: {} };
 
     it('throws on unhandled messages', () => {
       const view = ({ emit }) => (<button onClick={emit(Msg2)}>Foo</button>);
       const ctr = container({ name: 'FooContainer', update, view });
 
-      expect(() => shallow(ctr()).html()).to.throw(
+      expect(() => shallow(ctr(state)).html()).to.throw(
```

Comment:
Sorry, it didn't even occur to me before, but this actually has a pretty big impact on how tests are written, so `isolate()` will need to be patched to handle passing required parameters internally.